### ISSUE TITLE
Elements Missing ARIA Role on Before You File page

### ISSUE THREAD

**Issue by echappen** [ustaxcourt/ef-cms#596]
On [Before you File a Petition page](https://app.stg.ef-cms.ustaxcourt.gov/before-filing-a-petition), this element:

`<div class="grid-container padding-x-0 margin-bottom-5 create-case-steps" role="list">`

needs child elements with ARIA role of `listitem`

#### Solution
Add `role=“listitem”` to representative elements in this list, or remove `role=”list”` from parent element if not needed.

#### Why it Matters
To aid assistive technologies, elements with an ARIA `role` of `list` require children that represent items in that list. [Learn more](https://www.w3.org/TR/wai-aria-1.1/#list).

#### Notes
Issue found using [Lighthouse Accessibility Scan](https://developers.google.com/web/tools/lighthouse) on Chrome.

**Comment by JessicaMarine** [comment]
Moved to Flexion: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/8103
